code/game/objects/items/ego_weapons/aleph.dm

```diff
@@ -1120,3 +1120,117 @@
 	if(B.safety)
 		user.remove_status_effect(STATUS_EFFECT_LCBURN)
 
+/obj/item/ego_weapon/mockery
+	name = "mockery"
+	desc = "...If I earned a name, will I get to receive love and hate from you? \
+	Will you remember me as that name, as someone whom you emotionally cared for?"
+	special = "Use this weapon in hand to swap between forms. The whip has higher reach, the hammer deals damage in an area, and the bat knocks back enemies."
+	icon_state = "mockery_whip"
+	lefthand_file = 'icons/mob/inhands/64x64_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/64x64_righthand.dmi'
+	inhand_x_dimension = 64
+	inhand_y_dimension = 64
+	force = 35
+	attack_speed = 0.5
+	reach = 3
+	damtype = BLACK_DAMAGE
+	armortype = BLACK_DAMAGE
+	attack_verb_continuous = list("lacerates", "disciplines")
+	attack_verb_simple = list("lacerate", "discipline")
+	hitsound = 'sound/weapons/whip.ogg'
+	attribute_requirements = list(
+							FORTITUDE_ATTRIBUTE = 100,
+							PRUDENCE_ATTRIBUTE = 80,
+							TEMPERANCE_ATTRIBUTE = 80,
+							JUSTICE_ATTRIBUTE = 80
+							)
+	var/mob/current_holder
+	var/form = "whip"
+	var/list/weapon_list = list(
+		"whip" = list(35, 0.5, 3, list("lacerates", "disciplines"), list("lacerate", "discipline"), 'sound/weapons/whip.ogg'),
+		"sword" = list(80, 1, 1, list("tears", "slices", "mutilates"), list("tear", "slice","mutilate"), 'sound/weapons/fixer/generic/blade4.ogg'),
+		"hammer" = list(40, 1.4, 1, list("crushes"), list("crush"), 'sound/weapons/fixer/generic/baton2.ogg'),
+		"bat" = list(120, 1.4, 1, list("bludgeons", "bashes"), list("bludgeon", "bash"), 'sound/weapons/fixer/generic/gen1.ogg')
+		)
+
+/obj/item/ego_weapon/mockery/Initialize()
+	. = ..()
+	AddElement(/datum/element/update_icon_updates_onmob)
+
+/obj/item/ego_weapon/mockery/attack_self(mob/user)
+	. = ..()
+	if(!CanUseEgo(user))
+		return
+	SwitchForm(user)
+
+/obj/item/ego_weapon/mockery/equipped(mob/user, slot)
+	. = ..()
+	if(!user)
+		return
+	current_holder = user
+
+/obj/item/ego_weapon/mockery/dropped(mob/user)
+	. = ..()
+	current_holder = null
+
+/obj/item/ego_weapon/mockery/attack(mob/living/target, mob/living/carbon/human/user)
+	if(!CanUseEgo(user))
```

Comment:
/obj/item/ego_weapon already checks for CanUseEgo, consider removing this check

Reply:
Already being done.